packages/ui/src/components/Channel/Scroller.tsx

```diff
@@ -187,7 +190,7 @@ export default function Scroller({
       // (they are objects, not functions)
       const RenderItem = renderItem;
       return (
-        <View onLayout={() => handleItemLayout(item, index)}>
+        <View padding="$m" onLayout={() => handleItemLayout(item, index)}>
```

Comment:
Intentional? This adds extraneous padding around chat posts + would probably be better applied to the actual message or divider component getting rendered.

Reply:
Not intentional, was testing something, will fix.